osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs

```diff
@@ -34,11 +34,10 @@ public class DrawableLoungeRoom : DrawableRoom, IFilterable, IHasContextMenu, IH
         private const float transition_duration = 60;
         private const float selection_border_width = 4;
 
-        [Resolved(canBeNull: true)]
-        private LoungeSubScreen lounge { get; set; }
+        public readonly Bindable<Room> SelectedRoom = new Bindable<Room>();
```

Comment:
Feels a bit weird to still have this here. Might work better to move the "is selected" logic up one level (to `RoomsContainer`) and change this to 

```suggestion
        public readonly BindableBool IsSelected = new BindableBool(true);
```

Right now, the component can't be interacted with without `SelectedRoom` both being bound and being set to `true`, which is a bit of a hidden dependency.

Reply:
I didn't want to do this in order to to keep the changes small.

Reply:
I guess this line saves the flow from being completely broken:

https://github.com/ppy/osu/blob/d304e283e4a24506923f48097b84494086ef38d6/osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs#L162

Let's leave it for a follow-up change.